.github/workflows/docker-ci.yml

```diff
@@ -0,0 +1,73 @@
+name: docker-ci
+
+on:
+  push:
+    tags:        
+      - '*'
```

Comment:
This would collide with the Helm action tag: https://github.com/clastix/capsule/blob/7c1592e739838db822b317b0a6a356e39e2769c6/.github/workflows/helm.yml#L20

Can we just trigger this workflow when the tag starts with `v`?

Not sure if this would work:

```yaml
on:
  push:
    tags:
      - "v*"
```

Reply:
this line has been changed as requested inside commit [8473750](https://github.com/clastix/capsule/commit/8473750da9f362a4a74b76a5152d4f5d72648127)

everything works well